step.yml

```diff
@@ -63,39 +58,39 @@ inputs:
       is_required: true
   - upload_token:
     opts:
-      title: Waldo upload token
+      title: Upload token
       description: |
         Each application registered in Waldo is associated with a unique upload
         token.
       is_required: true
       is_sensitive: true
   - variant_name:
     opts:
-      title: "Waldo variant name"
+      title: "Variant name"
       description: |
         An arbitrary string that can be associated with a build.
       is_required: false
-  - find_symbols: "no"
+  - git_commit:
```

Comment:
You could use BITIRSE_GIT_COMMIT env var to get the git commit.
See https://devcenter.bitrise.io/builds/available-environment-variables/#exposed-by-bitriseio.